dev/io.openliberty.webcontainer.servlet.6.0.internal/src/io/openliberty/webcontainer60/session/impl/HttpSessionContextImpl60.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.webcontainer60.session.impl;
+
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+
+import java.util.logging.Level;
+
+import com.ibm.ws.session.SessionApplicationParameters;
+import com.ibm.ws.session.SessionManagerConfig;
+import com.ibm.ws.session.SessionStoreService;
+import com.ibm.ws.session.utils.LoggingUtil;
+import com.ibm.ws.webcontainer31.session.impl.HttpSessionContext31Impl;
+import com.ibm.wsspi.session.ISession;
+import com.ibm.wsspi.session.SessionAffinityContext;
+
+import io.openliberty.session.SessionData60;
+
+/**
+ * Since Servlet 6.0
+ */
+
+public class HttpSessionContextImpl60 extends HttpSessionContext31Impl {
```

Comment:
Does  HttpSessionContextImpl60 need to extend HttpSessionContext31Impl? or Can it extend HttpSessionContextImpl  (com.ibm.ws.webcontainer.session.impl.HttpSessionContextImpl) directly instead?

Reply:
I will keep this and try out later